config/locales/en.yml

```diff
@@ -3387,6 +3387,9 @@ See the %{link} to find out more about %{sitename}'s features and to start using
           producer_name: "Producer"
           unit: "Unit"
       general_settings:
+      shared:
+        sortable_header:
+          name: "Name"
```

Comment:
as per slack conversation, this is breaking general settings and also the other sortable_header entries, this needs to be moved to the correct place in a new PR

Reply:
Thank you @luisramos0, this my first PR has been a lovely chance to learn. Good thing is, I'm pretty sure I know exactly what happened. :-) Fixing this up now and will create a new PR